### PR TITLE
feat: Deploy goerli

### PR DESCRIPTION
**Pull request by fzavalia**

### CODE REVIEW

File: src/data/wearablesV1/addresses.ts [hunk 1]
```diff
@@ -45,10 +50,18 @@ export let RACBasics = '0x68e139552c4077ce5c9ab929c7e18ca721ffff00'
 export let WinklevossCapital = '0xc82a864a94db3550bc71fcb4ce07228bcec21f1a'
 export let DGAtariDillonFrancis = '0x51e0b1afe5da0c038fc93a3fc8e11cf7a238b40b'
 
-
 export function getCollectionsV1(): string[] {
+  let network = dataSource.network()
```
Comment (fzavalia) [inline src/data/wearablesV1/addresses.ts:54]: Without the network differentiation, mainnet would be counting test network collections (and test networks would be counting mainnet collections)

File: src/handlers/nft.ts [hunk 2]
```diff
@@ -170,9 +170,6 @@ export function handleAddItemV1(event: AddWearable): void {
     collection.isApproved = true
 
     collection.save()
-
-    let collectionMetric = buildCountFromCollection()
```
[the remark this answers is not in the document]
Comment (nachomazzara) [reply src/handlers/nft.ts:170]: Is this also correct for networks as Polygon and Mumbai where we don't have Ethereum collections?

[the remark this answers is not in the document]
Comment (fzavalia) [reply src/handlers/nft.ts:170]: This is happening in the `handleAddItemV1` function, which is executed each time a v1 collection emits the `AddWearable(indexed bytes32,string,uint256)` event.

[the remark this answers is not in the document]
Comment (fzavalia) [reply src/handlers/nft.ts:170]: Polygon and Mumbai collections never go through that handler

[the remark this answers is not in the document]
Comment (fzavalia) [reply src/handlers/nft.ts:170]: In more detail, ropsten, goerli and mainnet configs have a `address_fist_collection_v1`

That address is used to listen to the v1 collection `OwnershipTransferred` and handle it with the `handleInitializeWearablesV1`

The handler then initializes all hardcoded v1 collections by using the ERC721 template

```
export function handleInitializeWearablesV1(_: OwnershipTransferred): void {
  let count = buildCount()

  let collectionsV1 = getCollectionsV1()

  if (count.started == 0) {
    collectionsV1.forEach(collectionAddress => {
      // Create template bindings
      ERC721.create(Address.fromString(collectionAddress))
    })

    count.collectionTotal += collectionsV1.length
  }

  count.started = 1
  count.save()
}
```

This already sets the count for all v1 collections with just the ownership transferred of the first one.

**Review by nachomazzara — APPROVED** [review]

**Pull request merged**